test/ScanDependencies/ensure_no_astgen.swift

```diff
@@ -1,4 +1,5 @@
 // REQUIRES: swift_swift_parser
+// REQUIRES: asserts
```

Comment:
We ought to be able to run this test in both debug and non-debug builds. I assume we're seeing differing behavior here due to 5025964c2eb8d2a4622f619a6a5595aff45a4943, which only changed the release (with `LLVM_ENABLE_STATS` disabled) behavior to not print zero counters. We probably ought to add a flag to control whether zero counters are printed in both debug and release builds.

Reply:
Ok, how about `-print-zero-stats` then ?